test/test-debuglet.js

```diff
@@ -112,6 +113,7 @@ describe('Debuglet', function() {
 
       debuglet.once('registered', function(id) {
         assert.equal(id, DEBUGGEE_ID);
+        assert.equal(debuglet.debuggee_.project, projectId);
```

Comment:
For value comparisons I normally advocate for the use of `assert.strictEqual` to avoid 'truthy' equivocations but this is totally preference.